src/Resources/contao/library/Contao/File.php

```diff
@@ -681,32 +683,23 @@ public function resizeTo($width, $height, $mode='')
 	 */
 	public function sendToBrowser($filename=null)
 	{
-		// Make sure no output buffer is active
```

Comment:
I can create a separate PR for it so @leofeyer can merge it first and we then both rebase?